eclair-core/src/main/scala/fr/acinq/eclair/package.scala

```diff
@@ -44,6 +58,10 @@ package object eclair {
 
   def randomKey: PrivateKey = PrivateKey(randomBytes32)
 
+  def randomLong(): Long = {
```

Comment:
It made sense to put flat method calls in the package object when we just called basic methods on `secureRandom`, but now I think we should probably define a class that encapsulates all those xor calls, and hides `secureRandom` and `weakRandom` which are not supposed to be called directly.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1774/commits/0df44d9d5a3f6beac430640e213f8603185f57f0